src/GNDStk/type2node.hpp

```diff
@@ -31,6 +31,7 @@ and converting them into a node that's destined to be added as a child node.
    ) { \
       try { \
          node.clear(); \
+         node.name = "values"; \
```

Comment:
Why are you setting the node name to `values`?

Reply:
The GNDS content that's relevant to this code typically appears (or, for output, should appear) in a `<values>` node. Depending on just what somebody does to create a new node that they intend to write somewhere, the name might (and usually is) set elsewhere. And it *was* being so set - until I just recently wrote a direct test of this function. At that time, I realized that it would probably be a good idea to have the function put in something, not just `""`, for the default name.